### PR TITLE
fix: Respect the loglevel for logging

### PR DESCRIPTION
**Pull request by stphnrdmr**
This adds support for disabling console.log when the loglevel asks for it.

### CODE REVIEW

**Comment by stphnrdmr** [comment]
I think it could be argued that disabling this during `CI` is not universally loved. Respecting the log level seems like a good idea to me.

File: install.js [hunk 1]
```diff
@@ -145,3 +145,17 @@ function generateProtocolTypesIfNecessary(updated) {
     return;
   return require('./utils/protocol-types-generator');
 }
+
+function log(toBeLogged) {
+  function isTrue(value) {
+    return !!value && value !== '0' && value !== 'false';
+  }
+
+  const envDisable = isTrue(process.env.CI);
```
Comment (JoelEinbinder) [inline install.js]: Is it standard practice to disable logs when CI is set? I find the logs useful at least on our CI.

Comment (stphnrdmr) [reply install.js]: I agree with you and I’ll remove that part.

**Comment by stphnrdmr** [comment]
@JoelEinbinder Do you agree with the latest changes? I don't know why CI is failing right now.

File: install.js [hunk 2]
```diff
@@ -145,3 +145,12 @@ function generateProtocolTypesIfNecessary(updated) {
     return;
   return require('./utils/protocol-types-generator');
 }
+
+function log(toBeLogged) {
```
Comment (JoelEinbinder) [inline install.js]: nit: I'd rename log to logPolitely or to be a little easier to read what this does.

**Review by JoelEinbinder — APPROVED** [review]

File: install.js (hunk 2 above)
Comment (stphnrdmr) [reply install.js]: I like logPolitely, I’ll use that.

**Pull request merged**